x-pack/test/apm_api_integration/tests/transactions/breakdown.spec.ts

```diff
@@ -58,9 +87,9 @@ export default function ApiTest({ getService }: FtrProviderContext) {
 
         const { title, color, type, data, hideLegend, legendValue } = timeseries[0];
 
-        const nonNullDataPoints = data.filter((y: number | null) => y !== null);
+        const nonNullDataPoints = data.filter(({ y }: { y: number | null }) => y !== null);
 
-        expectSnapshot(nonNullDataPoints.length).toMatchInline(`61`);
+        expectSnapshot(nonNullDataPoints.length).toMatchInline(`47`);
```

Comment:
The assertion would remain the same if I change line 90 with 

```
const nonNullDataPoints = data.filter(({ x }: { x: number  }) => x !== null);
```
But since `x` is a number didn't make sense to me to check if it's null.